packages/arb-util/broadcastclient/broadcastclient.go

```diff
@@ -60,81 +60,103 @@ func NewBroadcastClient(websocketUrl string, lastInboxSeqNum *big.Int) *Broadcas
 	}
 }
 
-func (bc *BroadcastClient) Connect() (chan broadcaster.BroadcastFeedMessage, error) {
+func (bc *BroadcastClient) Connect(ctx context.Context) (chan broadcaster.BroadcastFeedMessage, error) {
```

Comment:
Looks like `Connect` can no longer return an error so it'd be good to simplify the return

Reply:
:+1: